src/include/sof/math/log_10.h

```diff
@@ -0,0 +1,20 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 Intel Corporation. All rights reserved.
+ *
+ * Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+ *
+ */
+
+#ifndef BASEE_10_LOG_H
+#define BASEE_10_LOG_H
+
+/* Include Files */
+#include "stdint.h"
+#include <stddef.h>
+#include <stdlib.h>
+#define BASE_10_COMMON_LOGARITM 0xA0000000
+
+/* Function Declarations */
+uint32_t log_10(uint32_t numerator, uint32_t denominator);
```

Comment:
we should probably just have a "log.h" so we don't liter the system with tiny files

Reply:
Done.